test/internal/utils.go

```diff
@@ -53,3 +57,14 @@ func (tc TestContext) UninstallOLM() {
 func (tc TestContext) KustomizeBuild(dir string) ([]byte, error) {
 	return tc.Run(exec.Command("kustomize", "build", dir))
 }
+
+// ReplaceInFile replaces all instances of old with new in the file at path.
+func ReplaceInFile(path, old, new string) {
```

Comment:
Where else is this being used? If it isn't revert this back to being unexported in Go e2e tests.

Reply:
In will revert it was added here to be used in both go tests. Now it will be part of a follow-up 👍